eng/testing/tests.mobile.targets

```diff
@@ -106,11 +106,30 @@
     <!-- This app is now can be consumed by xharness CLI to deploy on a device or simulator -->
   </Target>
 
+  <UsingTask TaskName="WasmAppBuilder"
+             AssemblyFile="$(ArtifactsBinDir)\WasmAppBuilder\$(MonoConfiguration)\$(NetCoreAppCurrent)\publish\WasmAppBuilder.dll" />
+
+  <Target Condition="'$(TargetOS)' == 'Browser'" Name="BundleTestWasmApp">
+	<ItemGroup>
+      <AssemblySearchPaths Include="$(PublishDir)"/>
+    </ItemGroup>
+    <Error Condition="!Exists('$(RuntimePackRidDir)')" Text="RuntimePackRidDir=$(RuntimePackRidDir) doesn't exist" />
+    <WasmAppBuilder
+      AppDir="$(BundleDir)"
+      RuntimePackDir="$(RuntimePackRidDir)"
+      MainAssembly="$(PublishDir)\WasmTestRunner.dll"
+      MainJS="$(MonoProjectRoot)\wasm\runtime-test.js"
+      ExtraAssemblies="$(PublishDir)\$(AssemblyName).dll"
+      AssemblySearchPaths="@(AssemblySearchPaths)"/>
+    <Exec Command="chmod a+x $(BundleDir)/run-v8.sh" />
```

Comment:
we can probably remove this since it will be handled by the WasmRunnerTemplate.sh